components/project-card/meta/title.jsx

```diff
@@ -13,18 +13,31 @@ class Title extends React.Component {
   }
 
   render() {
-    let classnames = classNames(`title mb-0 h4-heading`, this.props.className);
+    let content;
     let title = this.props.title;
+    let view = this.props.view;
+    let classnames = classNames(`title`, {
```

Comment:
We don't need to do this since the two sets of classes are mutually exclusive. We can just assign them to their corresponding wrapper div.

e.g., `<h4 className="h4-heading mb-0">{title}</h4>`